central/blob/datastore/datastore.go

```diff
@@ -10,6 +10,7 @@ import (
 
 // Datastore provides access to the blob store
 type Datastore interface {
+	GetNames(ctx context.Context) ([]string, error)
```

Comment:
I find this confusing as in the blob its the id and there is no name field. Can't we just call it get IDs